src/docbkx/chapter13-getting-started-monitor.xml

```diff
@@ -30,14 +30,65 @@
 		</para>
 		<mediaobject>
 			<imageobject>
-				<imagedata fileref="monitor_01_login.jpg" format="JPG"
-					scalefit="1" />
+				<imagedata fileref="monitor_01_login.jpg" format="JPG" scalefit="1" />
 			</imageobject>
 		</mediaobject>
 		<para>In the community edition of DataCleaner, you will find
 			suggestions for login credentials directly on the screen, to get you
 			started quickly.
 		</para>
+		<section>
+			<title>Creating a resources folder on Windows</title>
+			<para>
+				DataCleaner monitor uses a special resources folder. Typically it is created automatically during 
+				installation as <emphasis>C:\Windows\Users\USER_NAME\.datacleaner</emphasis>. However, when using 
+				Apache Tomcat and its system service then this resources folder needs to be created manually as 
+				<emphasis>C:\Windows\System32\config\systemprofile\.datacleaner</emphasis>. 
```

Comment:
Windows Explorer wont let you create any `.folder` names, so they need to do it from the command line. We'll probably need to explain how.